docs/installation/README_snowflake_datamigration.md

```diff
@@ -0,0 +1,350 @@
+# Data Migration Tool User Guide - Data Transfer
+* For Data Migration Tool introduction, please refer to [/README.md](/README.md)
+* For Installation user guide, please refer to [/docs/installation/README.md](/docs/installation/README.md)
+
+### Tool Features
+
+* Initial release will support bulk load data transfer only. 
+* Initial release will support Snowflake to BigQuery translation using BQ DTS, row and column validation of the data in source and target.
```

Comment:
says "Snowflake to BigQuery translation using BQ DTS", this is incorrect. Should be "BQ Translation service"